### DISCUSSION TITLE
Customize button widget

### DISCUSSION THREAD

**Discussion by ThinGlass** [rdbende/Sun-Valley-ttk-theme#54] [Q&A]
First of all, great theme! I especially like that is available on PyPi so it can be installed on the fly.
Second, I would like to add a border to a button, or something to make it stand out from the rest.
Is it possible to do that ?

**Comment by rdbende** [comment]
Adding some custom border is unfortunately not possible, but if you want a button that stands out from the rest, I recommend you to use the `Accent.TButton` style, which has a nice blue color.

```python
accent_button = ttk.Button(style="Accent.TButton")
```

**Reply by nicostoehr** [reply]
Hey rdbende, are there also other accents for other colors(red, ...)? Or is there a way to configure the style? Thanks

**Reply by rdbende** [reply]
No, unfortunately there aren't any other accent styles at the moment, and it's not possible to configure the colors dynamically.
